### ISSUE TITLE
Query view with stale set to ok will get nothing

### ISSUE THREAD

**Issue by nicolelovecoding** [pouchdb/pouchdb-server#253]
With the variable stale setting to ok, we will get nothing, but there are some docs in the db. If set the stale to be 'false' or 'true'. We can get something. For example, 
```
db.query('design_name/view_name', {
      stale: 'true',
      limit: limit,
      skip: skip,
      include_docs: true,
      reduce: false
    });

**Comment by gr2m** [comment]
@nicolelovecoding I guess `db` is instantiated with a URL which points to a PouchDB server? Or does the problem exist in PouchDB itself? Could you create a full code example which instantiates `db`, creates test documents, creates the view and runs the query? That will help us help you :)

**Comment by nicolelovecoding** [comment]
@gr2m 
Yes, db is instantiated with a URL which points to a local PouchDB server.
First start the abacus-pouchserver listening on port 5984
```
const start = (module) => {
      cp.spawn('npm', ['run', 'start'], {
        cwd: moduleDir(module),
        env: clone(process.env)
      });
    };
start('abacus-pouchserver');
```
Then create db and insert docs and view
In the end query view
```
var db = new PouchDB('http://admin:admin@localhost:5984/cf-names-0-201602');
```

**Comment by gr2m** [comment]
I haven’t seen abacus-pouchserver before, maybe https://github.com/cloudfoundry-incubator/cf-abacus would be a better place to file an issue on it?

**Comment by nicolelovecoding** [comment]
I also posted question there:
https://github.com/cloudfoundry-incubator/cf-abacus/issues/689#issuecomment-316488241
But abacus-pouchserver is just wrapper application on pouchserver. So I think here is the right place to ask question.
